lib/mutant/meta/example/dsl.rb

```diff
@@ -9,18 +8,20 @@ class DSL
         # Run DSL on block
         #
         # @return [Example]
-        def self.run(file, block)
+        def self.call(file, block)
```

Comment:
Because the call side has to capture the block into an object already, as it needs to be evaluated somewhere else. It does not make sense for me to pass it as a block to capture it to pass it as a block again.

Also a real argument counts against the arity checks, where a block does not.